index.html

```diff
@@ -12,18 +12,21 @@
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
 </head>
 <body>
-    <div class="wrapper">
+    <div id="wrapper">
         <div class="header">
-            <a href=""><img class="mobile_logo" src="./images/mobile_logo.png" alt="Mo.almeari Logo"></a>
+            <a href="#wrapper"><img class="mobile_logo" src="./images/mobile_logo.png" alt="Mo.almeari Logo"></a>
             <nav class="menu">
-                <div class="menu_icon">
-                    <a href=""><i class="material-icons">menu</i></a>
+                <div id="menu_container">
+                    <a href="#wrapper" class="close_menu"><img src="./images/x.png" alt=""></a>
+                    <ul class="menu_list">
+                        <li><a href="#works">Portfolio</a></li>
+                        <li><a href="#about">About</a></li>
+                        <li><a href="#contact">Contact</a></li>
+                    </ul>            
+                </div>
+                <div id="menu_icon">
+                    <a href="#wrapper"><i class="material-icons">menu</i></a>
                 </div>
-                <ul class="menu_list">
-                    <li><a href="">Portfolio</a></li>
-                    <li><a href="">About</a></li>
```

Comment:
Good Job @mohametalmeari 👲
- correct use of GitHub flow
- all linter checks passed
- your code so far is working as required, finish the remaining code and you're good to go  👍
- keep up the good work